### ISSUE TITLE
Default Google request token URL is invalid

### ISSUE THREAD

**Issue by opub** [3pillarlabs/socialauth#81]
The default request token URL for Google (https://www.google.com/accounts/o8/ud) returns a 404.

**Comment by sayantam** [comment]
Are you using the GooglePlus provider?

**Comment by opub** [comment]
Yes, I am now.  That one seems to be working.  The other Google provider should probably be removed if it no longer supported by Google.

**Comment by sayantam** [comment]
yes, it should be. i am leaving this open till we do this.
